src/formatter/helpers/keyword_helpers.js

```diff
@@ -0,0 +1,15 @@
+import { DIALECTS } from 'gherkin'
+
+/**
+ * In cases when a scenario is generated by a scenario outline
+ * the scenario's keyword equals to the scenario outline's keyword.
+ * The function replaces the scenario's keyword with the first keyword
+ * in the scenario keyword's list for the specified language.
+ */
```

Comment:
Took me a bit to understand this. Thoughts on the following:

```
If the keyword is one of the language's scenario outline keywords, 
  return the language's first scenario keyword
Otherwise returns the keyword
```